### PR TITLE
[feature/cumulative--10.0.0] AB#33500 - Close Inputs gap between Sketch Cloud and cm-ui library

### PR DESCRIPTION
**Pull request by fantaJinMode**
PR contains:
- Closing gap of `Prompt` component

### CODE REVIEW

File: docs/src/inputs/prompt/api/api.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import Main from '../../../global/main';
+/* eslint-disable import/no-named-default, import/extensions */
+import { default as rootDoc } from '!!@advclb/react-docgen-loader!react-cm-ui/inputs/prompt/prompt';
+/* eslint-enable import/no-named-default, import/extensions */
+
+const propTypes = {
+    location: PropTypes.shape({
+        pathname: PropTypes.string,
+    }).isRequired,
+};
```
Comment (morethanfire) [inline docs/src/inputs/prompt/api/api.jsx:16]: Please remove, location seems to be unused.

File: docs/src/inputs/prompt/prompt.jsx [hunk 2]
```diff
@@ -1,132 +1,110 @@
...
-                    Inline Prompt
-                    <Header.Subheader>
-                        A Button, Dropdown, or Link can have prompt attached to it asking for a confirmation from the end-user.
-                    </Header.Subheader>
-                </Header>
+                    <Example
+                        rawCode={require('!!raw-loader!./examples/exampleSuccessPrompt').default}
+                    >
+                        <ExampleSuccessPrompt />
+                    </Example>
```
Comment (morethanfire) [inline docs/src/inputs/prompt/prompt.jsx:61]: The JSX should be like...

```
<Main page={camelCase(displayName)}>
    <Main.Content>
        <MarkdownContainer>
            <Typography
                className="description"
                variant="body1"
            >
                {description}
            </Typography>
            
            <Heading
                anchorLink="solid-button"
                variant="h2"
            >
                Button with Alert Prompt
            </Heading>

            <Example
                rawCode={require('!!raw-loader!./examples/exampleFoo').default}
            >
                <ExampleFoo />
            </Example>
            
            <Heading
                anchorLink="solid-button"
                variant="h2"
            >
                Button with Warning Prompt
            </Heading>

            <Example
                rawCode={require('!!raw-loader!./examples/exampleBar').default}
            >
                <ExampleBar />
            </Example>
            
            <Heading
                anchorLink="solid-button"
                variant="h2"
            >
                Button with Informative Prompt
            </Heading>

            <Example
                rawCode={require('!!raw-loader!./examples/exampleBaz').default}
            >
                <ExampleBaz />
            </Example>
            
            <Heading
                anchorLink="solid-button"
                variant="h2"
            >
                Button with Confiming Prompt
            </Heading>

            <Example
                rawCode={require('!!raw-loader!./examples/exampleQux').default}
            >
                <ExampleQux />
            </Example>
        </MarkdownContainer>
        
        <ComponentVersionIdentifier
            pathname={pathname}
        />
    <Main.Content>
</Main>
```

Comment (morethanfire) [reply docs/src/inputs/prompt/prompt.jsx:61]: Resembling the mockup linked below:

https://www.sketch.com/s/6c8808fa-bf58-42e5-a0be-7e9d5798fc5f/a/VRRPro#Inspector

File: docs/src/inputs/prompt/examples/exampleAlertPrompt.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                <Button
+                    color="alert"
+                    disable={showPrompt}
+                    onClick={() => onPromptClick()}
+                    outlined
+                >
+                    Remove
+                </Button>
+            </Prompt>
+        </div>
```
Comment (morethanfire) [inline docs/src/inputs/prompt/examples/exampleAlertPrompt.jsx:42]: This needs all three button states shown as in:

image

File: docs/src/inputs/prompt/examples/exampleInformativePrompt.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        setShowPrompt(false);
+    }
+
+    function onNoClick() {
+        setShowPrompt(false);
+    }
+
+    return (
+        <div>
+            <Prompt
```
Comment (morethanfire) [inline docs/src/inputs/prompt/examples/exampleInformativePrompt.jsx:25]: This example needs all three states show as in: 

image

File: docs/src/inputs/prompt/examples/exampleSuccessPrompt.jsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        setShowPrompt(false);
+    }
+
+    function onNoClick() {
+        setShowPrompt(false);
+    }
+
+    return (
+        <div>
+            <Prompt
```
Comment (morethanfire) [inline docs/src/inputs/prompt/examples/exampleSuccessPrompt.jsx:25]: This example needs all three states show as in:

image

File: docs/src/inputs/prompt/examples/exampleWarningPrompt.jsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+        setShowPrompt(false);
+    }
+
+    function onNoClick() {
+        setShowPrompt(false);
+    }
+
+    return (
+        <div>
+            <Prompt
```
Comment (morethanfire) [inline docs/src/inputs/prompt/examples/exampleWarningPrompt.jsx:24]: This example needs all three states shown as in:

image

**Review by morethanfire — CHANGES_REQUESTED** [review]
Thank @sonurauniyar. Please go throught he comments and let me know if you have any questions.

File: docs/src/inputs/prompt/api/api.jsx (hunk 1 above)
Comment (fantaJinMode) [reply docs/src/inputs/prompt/api/api.jsx]: Fixed

File: docs/src/inputs/prompt/prompt.jsx (hunk 2 above)
Comment (fantaJinMode) [reply docs/src/inputs/prompt/prompt.jsx]: Fixed

File: docs/src/inputs/prompt/examples/exampleAlertPrompt.jsx (hunk 3 above)
Comment (fantaJinMode) [reply docs/src/inputs/prompt/examples/exampleAlertPrompt.jsx]: Fixed

File: docs/src/inputs/prompt/examples/exampleInformativePrompt.jsx (hunk 4 above)
Comment (fantaJinMode) [reply docs/src/inputs/prompt/examples/exampleInformativePrompt.jsx]: Fixed

File: docs/src/inputs/prompt/examples/exampleSuccessPrompt.jsx (hunk 5 above)
Comment (fantaJinMode) [reply docs/src/inputs/prompt/examples/exampleSuccessPrompt.jsx]: Fixed

File: docs/src/inputs/prompt/examples/exampleWarningPrompt.jsx (hunk 6 above)
Comment (fantaJinMode) [reply docs/src/inputs/prompt/examples/exampleWarningPrompt.jsx]: Fixed

**Review by morethanfire — APPROVED** [review]
Lookin' good! 👍

**Pull request merged**